optuna/integration/allennlp/_executor.py

```diff
@@ -165,17 +164,15 @@ def __init__(
         variable_manager = _VariableManager(target_pid)
 
         pruner_params = _fetch_pruner_config(trial)
-        pruner_params = {key: repr(value) for key, value in pruner_params.items()}
-
         variable_manager.set_value("study_name", trial.study.study_name)
-        variable_manager.set_value("trial_id", str(trial._trial_id))
+        variable_manager.set_value("trial_id", trial._trial_id)
         variable_manager.set_value("storage_name", url)
         variable_manager.set_value("monitor", metrics)
 
         if trial.study.pruner is not None:
             variable_manager.set_value("pruner_class", type(trial.study.pruner).__name__)
-            variable_manager.set_value("pruner_keys", DELIMITER.join(pruner_params.keys()))
-            variable_manager.set_value("pruner_values", DELIMITER.join(pruner_params.values()))
+            variable_manager.set_value("pruner_keys", list(pruner_params.keys()))
+            variable_manager.set_value("pruner_values", list(pruner_params.values()))
```

Comment:
How about just passing the `pruner_params` instead of dividing it into keys and values since they are re-constructed as `pruner_params` in the `_create_pruner`?